### PR TITLE
Resolve security flaw and bump dependency.

### LINKED ISSUE #414: Fix code scanning alert - Database query built from user-controlled sources
Tracking issue for:
- [ ] https://github.com/bubby932/VigorXRAPI/security/code-scanning/240

### PR DESCRIPTION
**Pull request by bubby932**
* Closes #414 (Resolves a security vulnerability in a developer endpoint, not critical, but important nonetheless.)
* Bumps `mongodb` as suggested by @dependabot.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "firebase-admin": "^11.2.0",
     "fuse.js": "^6.5.3",
     "jsonwebtoken": "^8.5.1",
-    "mongodb": "^4.10.0",
+    "mongodb": "^4.11.0",
```
Comment (bubby932) [inline package.json:27]: Package bumped by @dependabot.

File: routers/global.js [hunk 2]
```diff
@@ -19,15 +19,24 @@ router.route("/:key")
             res.sendStatus(404);
         }
     })
-    .post(middleware.authenticateDeveloperToken, async (req, res) => {
+    .patch(middleware.authenticateDeveloperToken, async (req, res) => {
```
Comment (bubby932) [inline routers/global.js:22]: Change HTTP verb from POST to PATCH to be less ambiguous.

File: routers/global.js [hunk 3]
```diff
@@ -19,15 +19,24 @@ router.route("/:key")
...
         const { mongoClient } = require('../index');
         try {
             const { key } = req.params;
             const { value } = req.body;
 
             const collection = mongoClient.db(process.env.MONGOOSE_DATABASE_NAME).collection("global");
 
-            collection.findOneAndUpdate({_id: key}, {data: value});
+            await collection.updateOne(
```
Comment (bubby932) [inline routers/global.js:30]: Uses a less wasteful & more explicit db query to resolve a security issue.

**Review by bubby932 — COMMENTED** [review]
Explained.

**Review by Bobrobot1 — APPROVED** [review]

**Comment by bubby932** [comment]
thaaaaaaaaaaaaaaaaaaaaaaaaaanks

**Pull request merged**